### PR TITLE
Add discv5 metrics to grafana dashboard

### PR DESCRIPTION
**Pull request by wemeetagain**
**Motivation**
See #3041 

**Description**

Feeds metrics from lodestar into discv5 when metrics are enabled
Adds current discv5 metrics to grafana dashboard under a "Discv5" row

### CODE REVIEW

File: packages/lodestar/src/metrics/metrics.ts [hunk 1]
```diff
@@ -66,3 +67,31 @@ export function createDbMetrics(): {metrics: IDbMetrics; registry: Registry} {
...
+      help: "Count of the discv5 connected peers",
+    }) as Gauge<string> & {collect(): void},
+    sentMessageCount: new Gauge<"type">({
+      name: "lodestar_discv5_sent_message_count",
+      help: "Count of the discv5 messages sent by message type",
+    }) as Gauge<"type"> & {collect(): void},
+    rcvdMessageCount: new Gauge<"type">({
+      name: "lodestar_discv5_rcvd_message_count",
+      help: "Count of the discv5 messages received by message type",
+    }) as Gauge<"type"> & {collect(): void},
```
Comment (g11tech) [inline packages/lodestar/src/metrics/metrics.ts]: labelNames: ["type"]

File: docker/grafana/provisioning/dashboards/lodestar.json [hunk 2]
```diff
@@ -7675,6 +7675,491 @@
...
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "lodestar_discv5_sent_message_count",
```
Comment (g11tech) [inline docker/grafana/provisioning/dashboards/lodestar.json]: would this be better as rate(lodestar_discv5_sent_message_count[$__rate_interval])?

File: docker/grafana/provisioning/dashboards/lodestar.json [hunk 3]
```diff
@@ -7675,6 +7675,491 @@
...
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "lodestar_discv5_rcvd_message_count",
```
Comment (g11tech) [inline docker/grafana/provisioning/dashboards/lodestar.json]: rate(lodestar_discv5_rcvd_message_count[$__rate_interval])

File: docker/grafana/provisioning/dashboards/lodestar.json [hunk 4]
```diff
@@ -7675,6 +7675,491 @@
...
+          "options": {
+            "dataLinks": []
+          },
+          "percentage": false,
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
```
Comment (g11tech) [inline docker/grafana/provisioning/dashboards/lodestar.json:8005]: may be these messages could be stacked as they would be labelled by type to show the clear spread & total volume

File: docker/grafana/provisioning/dashboards/lodestar.json [hunk 5]
```diff
@@ -7675,6 +7675,491 @@
...
```
Comment (g11tech) [inline docker/grafana/provisioning/dashboards/lodestar.json:7911]: stack?

**Review by dapplion — APPROVED** [review]

**Pull request merged**